cmd/geth/accountcmd_test.go

```diff
@@ -48,10 +48,16 @@ func tmpDatadirWithKeystore(t *testing.T) string {
 	return datadir
 }
 
+func runMinimalGethWithRaftConsensus(t *testing.T, args ...string) *testgeth {
+	argsWithRaft := append([]string{"--raft"}, args...)
+	return runMinimalGeth(t, argsWithRaft...)
+}
+
+/* unused anymore
```

Comment:
there is a choice between runMinimalGeth and runGeth to make.
Here is made the choice to adapt runMinimalGeth, but we can use the old runGeth.